### PR TITLE
Added implementation for Tizen.NET

### PR DESCRIPTION
**Pull request by JongHeonChoi**

### CODE REVIEW

File: src/TextToSpeech.Plugin.Tizen/TextToSpeech.cs [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+		static List<SupportedVoice> list;
+		uint maxLength = 0;
+
+		public TextToSpeech()
+		{
+			ttsInst = new TtsClient();
+			ttsInst.Prepare();
+			ttsInst.StateChanged += TtsStateChanged;
+			list = (List<SupportedVoice>)ttsInst.GetSupportedVoices();
+			Task.Delay(1000);
```
Comment (jamesmontemagno) [inline src/TextToSpeech.Plugin.Tizen/TextToSpeech.cs]: Any specific reason for this here? I usually have an "Init();" Type method that I call before speaking back.

File: src/TextToSpeech.Plugin.Tizen/TextToSpeech.cs [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+		static TtsClient ttsInst;
+		static List<SupportedVoice> list;
+		uint maxLength = 0;
+
+		public TextToSpeech()
+		{
+			ttsInst = new TtsClient();
+			ttsInst.Prepare();
+			ttsInst.StateChanged += TtsStateChanged;
+			list = (List<SupportedVoice>)ttsInst.GetSupportedVoices();
```
Comment (jamesmontemagno) [inline src/TextToSpeech.Plugin.Tizen/TextToSpeech.cs]: Could this ever throw null? Maybe us " as List<SupportedVoice>

File: src/TextToSpeech.Plugin.Tizen/TextToSpeech.cs [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+			}
+		}
+
+		/// <summary>
+		/// Get all installed and valid languages
+		/// </summary>
+		/// <returns></returns>
+		public Task<IEnumerable<CrossLocale>> GetInstalledLanguages()
+		{
+			return Task.FromResult(list.Select(a => new CrossLocale { Language = a.Language }));
```
Comment (jamesmontemagno) [inline src/TextToSpeech.Plugin.Tizen/TextToSpeech.cs:82]: Does seem like list could be null here?

**Review by jamesmontemagno — CHANGES_REQUESTED** [review]

**Comment by JongHeonChoi** [comment]
@jamesmontemagno @prashantvc Please, checking the code review.

**Pull request closed without merging**